app/assets/javascripts/sitewide/infinite_scroll.js

```diff
@@ -47,8 +47,6 @@ var InfiniteScroll = (function() {
 
     if (config.loadFirstPage) {
       loadData($container, 1);
-    } else if (scrollNotVisible($container)) {
```

Comment:
We don't need to increase default page size?

Reply:
I found, that this part generates a second request. First request correctly loaded elements.

Reply:
Yes, but is size of the first load enough to fill widget for all screen sizes?

Reply:
Yes, max-height for container 400px only 8 row can fit. We loaded 20 rows.